frappe/modules/utils.py

```diff
@@ -242,7 +242,7 @@ def make_boilerplate(template, doc, opts=None):
 
 		base_class = 'Document'
 		base_class_import = 'from frappe.model.document import Document'
-		if doc.is_tree:
+		if hasattr(doc, "is_tree"):
```

Comment:
```suggestion
		if doc.get('is_tree'):
```

Because it's possible that doc has `is_tree` attribute but its value might be 0 or false. In that case, the current check fails.